pom.xml

```diff
@@ -60,7 +60,7 @@
     <assertj.version>3.24.2</assertj.version>
     <awaitility.version>4.2.0</awaitility.version>
     <log4j.version>2.19.0</log4j.version>
-    <slf4j-api.version>2.0.4</slf4j-api.version>
+    <slf4j-api.version>1.7.36</slf4j-api.version>
```

Comment:
:crying_cat_face:

Reply:
:rofl: Yes. Log4j is not compatible yet. As a result, there were no logs printed anymore.